src/sentry/static/sentry/app/views/settings/components/forms/projectMapperField.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {components} from 'react-select';
+
+import InputField from 'app/views/settings/components/forms/inputField';
+import {ProjectMapperType} from 'app/views/settings/components/forms/type';
+import SelectControl from 'app/components/forms/selectControl';
+import IdBadge from 'app/components/idBadge';
+import Button from 'app/components/button';
+import {IconAdd} from 'app/icons';
+import {t} from 'app/locale';
+
+type Props = InputField['props'];
+type RenderProps = Props & ProjectMapperType;
+
+export class RenderField extends React.Component<RenderProps> {
```

Comment:
I'd call this a `ProjectMapperControl` where Field = Label + Control, and maybe for a follow-up PR when you get the designs.. I would move it into its own file.